scheduler/src/cook/config.clj

```diff
@@ -52,6 +53,12 @@
       'config/env-bool #(Boolean/valueOf (env %))}}
     config))
 
+(def passport-logger-ns "passport-logger")
+
+(defn log-passport-event
+  [log-data]
+  (log/log passport-logger-ns :info nil (json/write-str log-data)))
```

Comment:
Is `cook.config` the right namespace for this function? Maybe we should have a `cook.passport` namespace?

Reply:
Yeah, having a `cook.passport` namespace probably makes more sense. Added it in `cook.config` originally b/c that's where we create the logger but I agree that the `log-passport-event` function should have a different namespace